gridstatus/ercot.py

```diff
@@ -3171,7 +3171,11 @@ def parse_doc(
                 # So, it's N during DST And Y during Standard Time
                 # Pandas wants True for DST and False for Standard Time
                 # during ambiguous times
-                ambiguous = doc["DSTFlag"] == "N"
+                # Some ERCOT datasets use a boolean, some use a string
+                if doc["DSTFlag"].dtype == bool:
```

Comment:
there is another usage of DSTFlag in this function above. https://github.com/gridstatus/gridstatus/blob/cb77142c119651029847fc5af4be905141c9c2c7/gridstatus/ercot.py#L3147

perhaps we can define this `ambiguous` earlier in the function and reuse it multiple times

Reply:
Good idea. Added a separate method with an assertion.

https://github.com/gridstatus/gridstatus/pull/467/commits/017ca892eab4094d714a89895e3823b54ed97e7d